2.cellprofiler_processing/cputils.py

```diff
@@ -16,7 +16,9 @@ def rename_sqlite_file(sqlite_file_path: pathlib.Path, plate: str):
     """
     for sqlite_files in sqlite_file_path.iterdir():
         # the CellProfiler pipeline hardcodes the .sqlite file name, so all files outputted are called "CFReT.sqlite"
+        print(sqlite_files)
```

Comment:
generally we avoid having debugging print statements in production code unless it's important for logging.

Should we keep these print statements?

Reply:
No that was my bad, I will remove it. I was having some troubles but didn't realize it was still there.